x-pack/plugins/security/server/authentication/authentication_service.test.ts

```diff
@@ -739,7 +739,7 @@ describe('AuthenticationService', () => {
         expect(mockOnPreResponseToolkit.render).toHaveBeenCalledWith({
           body: '<div/>',
           headers: {
-            'Content-Security-Policy': `script-src 'unsafe-eval' 'self'; worker-src blob: 'self'; style-src 'unsafe-inline' 'self'`,
+            'Content-Security-Policy': `script-src 'self'; worker-src blob: 'self'; style-src 'unsafe-inline' 'self'`,
             Refresh:
               '0;url=/mock-server-basepath/login?msg=UNAUTHENTICATED&next=%2Fmock-server-basepath%2Fapp%2Fsome',
           },
```

Comment:
These authentication unit tests should not include any assertions about the CSP header, and you shouldn't have had to modify these tests in your PR.

This will happen again next time we change the default CSP. Can you take this opportunity to clean up this test and the others to remove such assertions? For example, in this case:

```diff
diff --git a/x-pack/plugins/security/server/authentication/authentication_service.test.ts b/x-pack/plugins/security/server/authentication/authentication_service.test.ts
index 3280bf91af8..535f28c1f36 100644
--- a/x-pack/plugins/security/server/authentication/authentication_service.test.ts
+++ b/x-pack/plugins/security/server/authentication/authentication_service.test.ts
@@ -738,11 +738,10 @@ describe('AuthenticationService', () => {
 
         expect(mockOnPreResponseToolkit.render).toHaveBeenCalledWith({
           body: '<div/>',
-          headers: {
-            'Content-Security-Policy': `script-src 'self'; worker-src blob: 'self'; style-src 'unsafe-inline' 'self'`,
+          headers: expect.objectContaining({
             Refresh:
               '0;url=/mock-server-basepath/login?msg=UNAUTHENTICATED&next=%2Fmock-server-basepath%2Fapp%2Fsome',
-          },
+          }),
         });
       });
```

Reply:
This might be a holdover from an earlier time in the "new platform" when Core wasn't providing the CSP header for us. In other words, I think I agree with you, but I'd like @azasypkin to double check my memory here

Reply:
> These authentication unit tests should not include any assertions about the CSP header, and you shouldn't have had to modify these tests in your PR.

I think it'd still be beneficial to validate that we do set CSP header in our custom view, what about test assertion like this?

```ts
 'Content-Security-Policy': CspConfig.DEFAULT.header,
```